src/credentials/vue/authForms/CredentialsForm.vue

```diff
@@ -0,0 +1,206 @@
+<template>
```

Comment:
Existing webviews/forms are not named "form", the `.vue` extension makes it clear enough. Using `feedback/vue/submitFeedback.vue` as a precedent, this would be named `editCredentials.vue` or something like that.

Reply:
I've re-opened a new PR since this was a draft.

What is the rationale for naming components verbs instead of nouns? I see a component as a piece of the page like a `footer` or `nav`.

> Existing webviews/forms are not named "form", the .vue extension makes it clear enough.
For example I have a component that is dedicated to being the title of a form since it is dynamic (can change to show an auth connected logo). Or a component that is a button. So the distinction seems necessary to me

Reply:
The stakes are admittedly low in either case :) but in general, YAGNI (i.e. "don't worry").

> For example I have a component that is dedicated to being the title of a form since it is dynamic (can change to show an auth connected logo). Or a component that is a button. So the distinction seems necessary to me

`title.vue` or `fooTitle.vue`, right? Doesn't necessarily need to be a verb and the lack of "form" in the name doesn't risk confusion (though it's still technically a form).

Moreover, this particular file already lives in `authForms/` (how many auth forms are we going to have? probably not many).